core/src/main/java/hudson/PluginManager.java

```diff
@@ -872,16 +872,16 @@ private boolean containsHpiJpi(Collection<String> bundledPlugins, String name) {
      * TODO: revisit where/how to expose this. This is an experiment.
      */
     public void dynamicLoad(File arc) throws IOException, InterruptedException, RestartRequiredException {
-        dynamicLoad(arc, false);
+        dynamicLoad(arc, false, null);
     }
 
     /**
      * Try the dynamicLoad, removeExisting to attempt to dynamic load disabled plugins
      */
     @Restricted(NoExternalUse.class)
```

Comment:
Note for reviewers. It was Restricted from very beginning, https://github.com/jenkinsci/jenkins/pull/2299